config/urls.py

```diff
@@ -12,6 +12,10 @@
 
 register_converter(SchemaFileConverter, "schemafile")
 
+handler404 = views.NotFoundView.as_view()
```

Comment:
These don't seem to get used?

Reply:
So the way these works in django is that when they are defined here, they are used over the defaults. If you check the code below, that's only set when in debug mode (which is useful for testing but not actually used in reality). If you don't set these explicitly, the normal django error views are used